core/schema/module.go

```diff
@@ -78,6 +78,9 @@ func (s *moduleSchema) Install(dag *dagql.Server) {
 		dagql.NodeFunc("generatedContextDirectory", s.moduleGeneratedContextDirectory).
 			Doc(`The generated files and directories made on top of the module source's context directory.`),
 
+		dagql.Func("localDefaults", s.moduleLocalDefaults).
+			Doc(`User-defined defaults values, loaded from local .env files.`),
```

Comment:
```suggestion
			Doc(`User-defined default values, loaded from local .env files.`),
```

re: trailing `.` - initially removed it, but we're super inconsistent there atm, to the point where I don't know what the pattern/guiding principle is anymore. For some reason I remember "first line has no `.`, subsequent ones do" but I don't remember if that came from solid ground.

Reply:
I have no clue either... I saw one with a dot, so did the same thing. No opinion either way.